lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -60,6 +60,22 @@ public static ObjectCannedACL getCannedAcl(S3FileSystemConfig.ObjectCannedAcl ca
         }
     }
 
+    public enum RetryMode
+    {
+        STANDARD,
+        LEGACY,
+        ADAPTIVE;
+
+        public static software.amazon.awssdk.core.retry.RetryMode getRetryMode(RetryMode retryMode)
```

Comment:
nit: this could be the field in the enum

Reply:
right, but this pattern was requested instead when I did ObjectCannedACL. So being consistent here